components/places/sql/create_shared_temp_tables.sql

```diff
@@ -44,3 +45,45 @@ CREATE TEMP TABLE moz_updateoriginsupdate_temp (
     PRIMARY KEY (prefix, host)
 ) WITHOUT ROWID;
 
+-- Tags
```

Comment:
Why are these temp triggers now here? I'd expect to see them in `create_shared_triggers.sql`

Reply:
Yep, I pasted them into the wrong file! 📋 🍝